app/controllers/mixins/actions/host_actions/misc.rb

```diff
@@ -158,6 +169,8 @@ def process_hosts(hosts, task, display_name = nil)
           when 'manageable'         then process_hosts_manageable(hosts, display_name)
           when 'introspect'         then process_hosts_introspect(hosts, display_name)
           when 'provide'            then process_hosts_provide(hosts, display_name)
+          when 'check_compliance_queue', 'scan_and_check_compliance_queue'
+            process_hosts_compliance(hosts, task, display_name)
```

Comment:
NOTE check_compliance_queue and scan_and_check_compliance_queue still use AvailabilityMixin and this is used by more than just the Host model so not as simple a transition